src/core/websocket/ws.ts

```diff
@@ -22,6 +23,7 @@ export type ErrorEvent = Events.ErrorEvent;
 export type CloseEvent = Events.CloseEvent;
 
 export type Options = {
+    accessToken?: string;
```

Comment:
Add note/TODO to mention that this is only available in websocket options for now